hms-app/src/pages/Voting.tsx

```diff
@@ -188,7 +187,9 @@ export default function Voting() {
                             {column.items.map((item, index) => {
                               return (
                                 <Draggable
+                                  // @ts-ignore
```

Comment:
I couldn't go through it yesterday, but the reason that you need to add those ignores, is unclear structure of your variables which can lead to - undefined, errors and other bad things. The best option would be to never add such annotations and use strong typing. 😄 

You were missing, at the def. level:
```
const columnsFromBackend : { [key: string]: {name: string, items: {id: string}[]}; }= {
  ['1']: {
    name: 'All ideas',
    items: [],
  },
  ['2']: {
    name: 'Your Votes',
    items: [],
  },
}
```

Reply:
or simpler like:
```
type Item = {id: string}

const columnsFromBackend = {
  ['1']: {
    name: 'All ideas',
    items: [] as Item[],
  },
  ['2']: {
    name: 'Your Votes',
    items: [],
  },
}
```